ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.scss

```diff
@@ -0,0 +1,52 @@
+$border-color: #EBEBEB;
+$icon-width: 24px;
+$icon-margin-right: 2rem;
+$question-padding: 1.5rem;
+
+.faq-block__icon {
+    font-size: 30px;
+    margin-right: $icon-margin-right;
+}
+
+.faq-block__question {
+    margin: 0;
+}
+
+.faq-block__text {
+    white-space: pre-wrap;
+}
+
+.mat-expansion-panel {
+    border-top: 2px solid $border-color;
+    font-family: Untitled-Sans-Regular;
+    font-size: 24px;
+
+    &:last-child {
+        border-bottom: 2px solid $border-color;
+    }
+
+    :hover, :focus {
+        background-color: rgba(0,0,0,.04);
```

Comment:
it's very strange we need to add this styles manually. Angular material already provides the styles for hover and focus, you can check here
https://material.angular.io/components/expansion/overview
maybe we need to inject material styles?

Reply:
Agree, it was a workaround. 
Added angular prebuilt theme for now (cadb3b5b36f72cc5fdcb7095cecbb9c1e9aca920).